test/images/scripts/lib/make-srpm

```diff
@@ -11,21 +11,13 @@ if [ -z "$version" ]; then
 fi
 
 modify_spec() {
-case $version in
-wip)
-sed '1i\
-%define gitcommit wip'
-;;
-*)
-sed "1i\
-%define tag $version"
-esac
+sed -e "1i\
+%define gitcommit wip\n%define tag $version\n"
```

Comment:
> @mvollmer Do all the images need to be regenerated when files here change?

`make-srpm` is called by default on every image creation and used inside the setup script for build/mock config.